app/views/invoices/_invoice_body.html.slim

```diff
@@ -13,7 +13,11 @@ table.table.invoice class="#{'table-bordered' if request.format.html?}"
       td.word-break
         |Bank account for wire transfers:<br>
         = current_member.user_full_name
-        p= current_organization.bank_accounts.visible.by_currency(invoice.currency).first.try(:invoice_details)
```

Comment:
@yunixon there is `invoice_details` method in `Invoice` model. Update it and use here.

Reply:
@shir we haven't invoice_details in Invoice model